sklearn/covariance/graph_lasso_.py

```diff
@@ -461,8 +461,8 @@ class GraphLassoCV(GraphLasso):
         values of alphas are passed.
 
     cv : cross-validation generator, optional
-        see sklearn.cross_validation module. If None is passed, defaults to
-        a 3-fold strategy
+        see sklearn.model_selection.split module.
```

Comment:
Yes, I mean to say `Stratified` if a similar docstring occurs in a classifier context. By the way, @jnothman, what do you think about removing the `classifier` argument completely? [Discussion here](https://github.com/scikit-learn/scikit-learn/pull/4294/files#r33638228)